contacts.install

```diff
@@ -17,18 +17,28 @@ function contacts_requirements($phase) {
     $themeHandler = \Drupal::service('theme_handler');
 
     if (!$themeHandler->themeExists('contacts_theme')) {
-      // Try to install theme.
-      $success = \Drupal::service('theme_installer')->install(['contacts_theme']);
-
-      if (!$success) {
-        $requirements['theme'] = array(
-          'title' => t('Missing Theme'),
-          'value' => 'To enable the contacts module you must first enable the contacts theme.',
-          'severity' => REQUIREMENT_ERROR,
-        );
-      }
+      $requirements['contacts_theme_missing'] = array(
```

Comment:
Think you need to use $theme_data = \Drupal::service('theme_handler')->rebuildThemeData() to see if a theme is available (see \Drupal\Core\Extension\ThemeInstaller::install).